app/models/manageiq/providers/ansible_tower/configuration_manager/credential.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::Credential < Authentication
+end
```

Comment:
Yes. I am going to work on the test once we are more or less agreed on the modeling.